### DISCUSSION TITLE
Can't get unlocker working

### DISCUSSION THREAD

**Discussion by Weezerxiv14** [taligentx/dscKeybusInterface#341] [dscKeybusInterface Discussion]
Hi I can't get unlocker working. Anyone else having the same issue? Tied with uno, Mega2560 and esp32 on a Powerseries Pc1616.

Thank you

**Comment by kaithefinn** [comment]
Hi Weezerxiv14,
for better understanding your issue, did you solder the circuit board with the resistors and one transistor and connected that properly to your DSC alarm system (parallel to your DSC panel) and on the other hand to your Arduino Uno (pins 3,5,6). If you need more help in this I can send you photographs and an easy circuit board layout, that I did by my own and it works well.
Kind regards,
Kaithefinn
